mmv1/products/datastream/terraform.yaml

```diff
@@ -93,6 +93,72 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       constants: templates/terraform/constants/private_connection.go.erb
       post_create: templates/terraform/post_create/private_connection.go.erb
       post_import: templates/terraform/post_import/private_connection.go.erb
+  Stream: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: projects/{{project}}/locations/{{location}}/streams/{{stream_id}}
+    import_format: ["projects/{{project}}/locations/{{location}}/streams/{{stream_id}}"]
+    properties:
+      destinationConfig.gcsDestinationConfig.fileRotationInterval: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+      destinationConfig.destinationConnectionProfile: !ruby/object:Overrides::Terraform::PropertyOverride
+        # API currently returns an id that uses a number instead of a project id
+        ignore_read: true
+      sourceConfig.sourceConnectionProfile: !ruby/object:Overrides::Terraform::PropertyOverride
+        # API currently returns an id that uses a number instead of a project id
+        ignore_read: true
+      sourceConfig.mysqlSourceConfig.maxConcurrentCdcTasks: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+        validation: !ruby/object:Provider::Terraform::Validation
+          function: 'validation.IntAtLeast(0)'
+    virtual_fields:
+      - !ruby/object:Api::Type::Enum
+        name: 'desired_state'
```

Comment:
Does it mean `state` is partially client and partially server specified field?

Reply:
yes